### ISSUE TITLE
Add visual to collection list for when mods override each other

### ISSUE THREAD

**Issue by Arkhorse** [bcssov/IronyModManager#381]
Bit difficult to explain, however this system would show when one mod has code or files that overrides another. This idea is from [Mod Organizer 2](https://github.com/ModOrganizer2/modorganizer), which uses this to parse the plugins to show when one mods records override another, with the ability to also show when packed and loose files override.

This would need to properly handle FIOS/LIOS/Override handles and parse all files within a mod. Doing this on demand might be resource intensive, so indexing mods may be a good way around that. Should have settings related to ignoring file types and/or folders. Would most likely want to blacklist the `gfx` folder by default to reduce the resource usage. 

This would allow people to create collections without meaningful conflicts without needing to run the conflict manager. 

Would be classified as nice to have

**Comment by bcssov** [comment]
Since I've never used this particular program and I'm not sure what to look for I'll need ELI5.

**Comment by bcssov** [comment]
Had someone explain it to me. This is too taxing on the system, have to reject. Plus this conflicts with a lighter version of this task: #20
